core/loadpoint_effective.go

```diff
@@ -76,7 +76,13 @@ func (lp *Loadpoint) nextVehiclePlan() (time.Time, int, int) {
 
 // EffectivePlanSoc returns the soc target for the current plan
 func (lp *Loadpoint) EffectivePlanSoc() int {
+	if lp.planActive {
+		return lp.planActiveSoc
+	}
+
 	_, soc, _ := lp.nextVehiclePlan()
+	lp.planActiveSoc = soc
```

Comment:
Diese implizite Zustandsveränderung in einem Getter ist nicht gut und von außen nicht erwartbar.
Das Setzen und Zurücksetzen von `lp.planActiveSoc` und `lp.planActiveTime` sollte immer in Sync mit `lp.planActive` passieren. Ggf. können wir die Notwendigkeit für `planActive` sogar ersetzen bzw. daraus ableiten ob es eine `planActiveTime` gibt.